src/core_modules/capture-core/components/EventWorkingListsCommon/useDefaultColumnConfiguration/useDefaultColumnConfig.js

```diff
@@ -1,17 +1,18 @@
 // @flow
 import { useMemo } from 'react';
+import { translatedStatusTypes } from 'capture-core/metaData/DataElement/statusTypes';
```

Comment:
actually I was using it in [different PR](https://github.com/dhis2/capture-app/pull/1788/files#diff-6aa51b62b0c8fabcb14784199f74f8a778c01c921c46381bb1498a1e386de0b3)
so I want to create same file to reuse it

Reply:
Ok, sounds good 👍  I would put it somewhere else then in the metadata folder though. Maybe in the events folder (capture-core/events) because we're talking about event statuses here.